roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
```

Comment:
Actually I am ok with offered way. But also note that  putting  configs to mantl-storage-setup mainly spreaded by different roles only for logic reasosn and convinience. It designed to run on first-boot really, as docker.service dependency.  Also would need add mesos.service's wanted-by as well to storage setup service, as well as to script itself to add it to generated .mount files.